librz/il/core_theory_vm.h

```diff
@@ -0,0 +1,143 @@
+#ifndef BUILD_CORE_THEORY_VM_H
+#define BUILD_CORE_THEORY_VM_H
+
+#include "definitions/wrapper.h"
+#include "core_theory_opcodes.h"
+
+// TODO replace with rz hash table
```

Comment:
I think it was done already, no?

Reply:
Yes it was done, I have removed some out-of-date TODOs in the local commmit.